build-container-dev.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+#
+# Copyright (c) 2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+#
+# Copyright 2019-2020 JetBrains s.r.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -e # Any command which returns non-zero exit code will cause this shell script to exit immediately
+set -x # Activate debugging to show execution details: all commands will be printed before execution
+
+cd ../projector-server
```

Comment:
hello, I think it does not work well with absolute path

```
base_dir=$(cd "$(dirname "$0")"; pwd)
cd "${base_dir}"/../projector-server
```
but wondering if we could not just call gradlew from the other folder 
```
${base_dir}"/../projector-server/gradlew --console=plain :projector-server:distZip
```

so we avoid cd, cd - (pushd/popd is more robust)

Reply:
Calling gradlew by absolute path without cd command into projector directory produces build error:
```
Project 'projector-server' not found in root project 'jetbrains-editor-images'.
```
Gradlew tries to find the sources from the current working directory.

Added `base_dir` instruction.